tests/puppeteer/pages/BO/orders/view.js

```diff
@@ -27,9 +27,9 @@ module.exports = class Order extends BOBasePage {
     this.documentNumberLink = '#documents_table tr:nth-child(%ID) td:nth-child(3) a';
     this.documentName = '#documents_table tr:nth-child(%ID) td:nth-child(2)';
     // Refund form
-    this.refundProductQuantity = `${this.orderProductsTable} tr:nth-child(%ID) 
+    this.refundProductQuantity = `${this.orderProductsTable} tr:nth-child(%ID)
```

Comment:
The string is written on two lines? Wouldn't it be better on a single line?

Reply:
`max-len = 120` for eslint, and we have an error if we make the selector in a single line

Reply:
That's what I figured, I just thought the syntax was weird, but as long as as it works correctly